src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
 
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
+        }
+    }
```

Comment:
The messages in the `doWork` and `stopWork` methods should be informative and indicate what type of `Machine` is working or stopping. Ensure that the implementation of these methods in the `Truck`, `Bulldozer`, and `Excavator` classes provide clear messages reflecting the type of machine.